packages/utils/src/getFeatureFlag.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Analytics } from '@deriv-com/analytics';
+
+const waitForGrowthbook = () => {
+    return new Promise(resolve => {
+        const checkAnalytics = () => {
+            if (typeof Analytics !== 'undefined' && Analytics.getInstances()?.ab !== undefined) {
```

Comment:
does this check for some requests to actually be done (so does it have potential to never be fulfilled), or does it only check for the presence of Analytics instance? 
basically wondering if we need a timeout and error hadnling as fallback in this function,

Reply:
So the initialization of Analytics is coming from another file and I need to wait for that to happen, having analytics ready is not enough for us, we need to also make sure it's initialized.

Reply:
yes, but the question is about analytics erroring out, 
if the analytics / GB fails, will it cause the whoe app to fail?

Reply:
In the new change it I made with discussion with @vinu-deriv , there is a possibility that the promised won't get resolved and thus we have added a 10seconds buffer to auto resolve it no matter what. Tested it with invalid growthbook keys and this check will only need to occur once, on page load.